aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/AWSS3StoragePlugin.java

```diff
@@ -107,7 +108,7 @@ public String getPluginKey() {
 
     @Override
     @SuppressWarnings("MagicNumber") // TODO: Remove once default values are moved to configuration
-    public void configure(@NonNull JSONObject pluginConfiguration, @NonNull Context context) throws StorageException {
+    public void configure(@Nullable JSONObject pluginConfiguration, @NonNull Context context) throws StorageException {
```

Comment:
It isn't; passing a null value here will result in a `StorageException`. This should say `@NonNull`.